test/e2e/framework/pod/create.go

```diff
@@ -0,0 +1,312 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pod
+
+import (
+	"fmt"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/uuid"
+	clientset "k8s.io/client-go/kubernetes"
+	e2elog "k8s.io/kubernetes/test/e2e/framework/log"
```

Comment:
Could you update to use `framework.Logf()` instead of `e2elog.Logf()` because of https://github.com/kubernetes/kubernetes/issues/81427 ?

Reply:
ping @timothysc @andrewsykim for consensus.
Just want to double check on this. I didn't hear/see about the decision to move the functionality from e2elog back to the framework

Reply:
Looking at the conversation from https://kubernetes.slack.com/archives/C9NK9KFFW/p1565813776113500?thread_ts=1565737650.112200&cid=C9NK9KFFW
it seems like the `framework.Logf()` was suggested as an option in the case of dependency problems (?)

Reply:
@oomichi coming back to https://github.com/kubernetes/kubernetes/issues/81427 (using `framework.logF()` for tests `e2elog.LogF` for framework subpkgs.
Taking care of this type of PR is messy - a lot of rebasing as we touch a ton of files during refactorings. What do you think about getting this one through the door and putting me down for a cleanup of the `framework/pod` pakcage as a follow up?

Reply:
No problem if we can follow up it later with another PR.
I just wanted to let you know that at the first time :-)

Reply:
I didn't think you were going to be so fast at replying :sweat_smile: 
could you take a look again, please

And will definitely follow up with the required changes for logging.